{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests58_37_2.py::test_matrix_of_one_integer", "location": ["tests58_37_2.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_37_2.py": 1, "Folder_37": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00017425499390810728, "start": 1700336881.998538, "stop": 1700336881.998713, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "a212b03d188048be8ae3cb137d821c09", "node": "<WorkerController gw0>"}
{"nodeid": "tests58_37_2.py::test_horizontal_matrix", "location": ["tests58_37_2.py", 40, "test_horizontal_matrix"], "keywords": {"test_horizontal_matrix": 1, "__wrapped__": 1, "tests58_37_2.py": 1, "Folder_37": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001471450086683035, "start": 1700336882.0083113, "stop": 1700336882.008459, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "a212b03d188048be8ae3cb137d821c09", "node": "<WorkerController gw1>"}
{"nodeid": "tests58_37_2.py::test_matrix_of_one_integer", "location": ["tests58_37_2.py", 18, "test_matrix_of_one_integer"], "keywords": {"test_matrix_of_one_integer": 1, "__wrapped__": 1, "tests58_37_2.py": 1, "Folder_37": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[64]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 64 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_results_2/Folder_37/tests58_37_2.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_mat[k-i][l-j] = matrix[k][l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_results_2/Folder_37/generated_answer.py", "lineno": 18, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[64]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_one_integer():", "        input_matrix = [[64]]", "        input_matrix = np.asmatrix(input_matrix)", "        if 64 == 1:", "            assert submatrix_with_n_numbers(input_matrix) == 1", "        else:", ">           assert not submatrix_with_n_numbers(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_results_2/Folder_37/tests58_37_2.py", "lineno": 26, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    temp_mat[k-i][l-j] = matrix[k][l]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q58/codellama_results_2/Folder_37/generated_answer.py", "lineno": 18, "message": "in submatrix_with_n_numbers"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[64]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0007178829982876778, "start": 1700336881.9989772, "stop": 1700336881.9996955, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "a212b03d188048be8ae3cb137d821c09", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
